cumulusci/tasks/salesforce/users/permsets.py

```diff
@@ -29,65 +31,98 @@ def _init_options(self, kwargs):
         super()._init_options(kwargs)
 
         self.options["api_names"] = process_list_arg(self.options["api_names"])
+        self.options["user_alias"] = process_list_arg(
+            self.options.get("user_alias") or []
+        )
 
     def _run_task(self):
-        # Determine existing assignments
-        if "user_alias" not in self.options:
+        users = self._query_existing_assignments()
+        users_assigned_perms = {
+            user["Id"]: self._get_assigned_perms(user) for user in users
+        }
+        self.perms_by_id = self._get_perm_ids()
```

Comment:
I don't feel strongly about it to argue for a change, but the mixing between local parameters and the instance var confused me on first read.